src/mewbot/tools/lint.py

```diff
@@ -38,14 +39,24 @@ class LintToolchain(ToolChain):
     module or any installed plugins - are linted.
     """
 
+    fail_ci_run: bool = True
+
     def run(self) -> Iterable[Annotation]:
         """Runs the linting tools in sequence."""
 
-        yield from self.lint_black()
-        yield from self.lint_flake8()
-        yield from self.lint_mypy()
-        yield from self.lint_pylint()
-        yield from self.lint_pydocstyle()
+        with CommandDeliminator("Starting black tool run"):
+            yield from self.lint_black()
+        with CommandDeliminator("Starting flake8 tool run"):
+            yield from self.lint_flake8()
+        with CommandDeliminator("Starting mypy tool run"):
+            yield from self.lint_mypy()
+        with CommandDeliminator("Starting pylint tool run"):
+            yield from self.lint_pylint()
+        with CommandDeliminator("Starting pydocstyle tool run"):
+            yield from self.lint_pydocstyle()
+
+        if self.is_ci and self.fail_ci_run:
+            print(f"Some of the tools reports having silently failed!\n{self.run_success}")
```

Comment:
What's stopping this from appearing if something hasn't silently failed? I cannot see where that would occur.

Reply:
This has been removed :)